docs/components/popover.md

```diff
@@ -221,6 +221,68 @@ export default {
 </script>
 ```
 
+### PositionFixed
```

Comment:
I believe this prop title is fine; however, I noticed `buttonText` prop title on this page is camelCase while the rest are PascalCase. 

Can you update `buttonText` to `ButtonText` with this PR?

Reply:
Switched everything to `camelCase` 🐫